examples/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.9.5)
+project(ITKMontageExamples)
+
+find_package(ITK REQUIRED
+  COMPONENTS ITKImageIO ITKTransformIO Montage
+  )
+include(${ITK_USE_FILE})
+find_package(ITK REQUIRED
```

Comment:
Again find_package?

Reply:
Yes, I did not mean to commit this -- good catch!

Reply:
@fbudin69500 and this is the cause of the "issue" I reported above -- nevermind! :put_litter_in_its_place: